src/axom/mint/docs/sphinx/index.rst

```diff
@@ -3,14 +3,88 @@
 .. ##
 .. ## SPDX-License-Identifier: (BSD-3-Clause)
 
-================
 Mint User Guide
-================
+===============
 
- .. include:: sections/synopsis.rst
```

Comment:
Was the include a problematic here?

Reply:
It's subtle... but here is the difference:

https://axom.readthedocs.io/en/develop/axom/slic/docs/sphinx/index.html

https://axom.readthedocs.io/en/feature-white238-unifysphinxtoc/axom/slic/docs/sphinx/index.html

Reply:
I see -- Thanks for clarifying.